src/components/organisms/pupil/OakPupilJourneyListItemSubheading/OakPupilJourneyListItemSubheading.tsx

```diff
@@ -1,27 +1,24 @@
 import React from "react";
 
-import { OakPupilJourneyListCounter } from "../OakPupilJourneyListCounter";
-
 import { OakBulletList, OakBulletListProps } from "@/components/molecules";
 import { OakFlex } from "@/components/atoms";
 
-export type OakPupilJourneyPreviousLessonsProps = {
-  numberOfLessons: number;
+export type OakPupilJourneyListItemSubheadingProps = {
+  textSlot?: React.ReactNode;
```

Comment:
I'm a bit concerned that we might be "over-composing" our components (sorry can't think of a better term). Does this really need a dedicated component ? And if so does it really need to employ composition for the textSlot (ie. is it ever not an OakPupilJourneyListCounter )